blog/models.py

```diff
@@ -18,3 +19,46 @@ def publish(self):
 
     def __str__(self):
         return self.title
+
+
+class CLIP(models.Model):
+
+    bar_choices = [
+        ("barcode_set1.csv", "barcode_set1.csv"),
+    ]
+    chrom_choices = [
+        ("hg38.chrom.sizes", "hg38.chrom.sizes"),
+        ("hg19.chrom.sizes", "hg19.chrom.sizes"),
+    ]
+    star_choices = [
+        ("star_2_7_gencode29_sjdb", "star_2_7_gencode29_sjdb"),
+        ("star_2_7_6a_release6_sjdb", "star_2_7_6a_release6_sjdb"),
+    ]
+
+    description = models.TextField(max_length=200, blank=True)
+    barcode_file = models.CharField(max_length=25, choices=bar_choices, blank=True)
+    chrom_sizes = models.CharField(max_length=50, choices=chrom_choices, blank=True)
+    star_index = models.CharField(max_length=50, choices=star_choices, blank=True)
+    umi_pattern = models.CharField(max_length=10, default="NNNNNNNNNN")
+    # set default: 10 Ns
+    fastqs = models.CharField(max_length=200, blank=True)
+
+    def __str__(self):
+        return self.barcode_file
+
+
+class Fastq(models.Model):
+
+    adapt_choices = [
+        ("InvRiL19_adapters.fasta", "InvRiL19_adapters.fasta"),
+        ("InvRNA1.fasta", "InvRNA1.fasta"),
+        ("InvRNA2.fasta", "InvRNA2.fasta"),
+    ]
+
+    title = models.CharField(max_length=100)
+    path = models.CharField(max_length=100)
+    adapter_path = models.CharField(max_length=50, choices=adapt_choices, blank=True)
```

Comment:
In my yaml_feature_1 branch I have this in my CLIP model and is made to be a dropdown in the form. Is there a reason you moved this to the Fastq model? It looks like you're trying to create a dropdown menu for the adapter_path field. Since we don't define Fastq as a ModelForm in forms.py, on the form it just looks like a normal text box and not a dropdown.

Reply:
To make this into a dropdown, we could perhaps move this back to the CLIP model and change the CLIP_yaml() method accordingly in views.py

But if you want to keep it as a Charfield we can just remove `choices=adapt_choices`

Reply:
Good question. Each fastq file will need its own set of adapters (see attached 'real' example), so I moved the adapters field out and into the Fastq object. 

 [example_clip.txt](https://github.com/YeoLab/metadata-db/files/9501419/example_clip.txt)

Reply:
The dropdown is nice for users who can select adapters, what we can do here is commit/merge these changes if they improve the code, and open an issue to add this 'feature' in?